stories/atoms/components/fileInput/FileInput.stories.mdx

```diff
@@ -0,0 +1,479 @@
+import { ArgsTable, Meta, Story, Canvas } from '@storybook/addon-docs'
+import { FileInput } from 'ui/atoms/fileInput/FileInput'
+import { Typography } from 'ui/atoms/typography/Typography'
+import { ThemeProvider } from 'ui/atoms/theme/ThemeProvider'
+import { ThemeSwitcher } from 'ui/atoms/theme/ThemeSwitcher'
+import { useState } from 'react'
+
+<Meta
+  title="Atoms/FileInput"
+  component={FileInput}
+  parameters={{
+    actions: { argTypesRegex: '^on.*' },
+    docs: {
+      source: {
+        type: 'code',
+        format: true,
+        language: 'jsx'
+      }
+    }
+  }}
+  decorators={[
+    Story => (
+      <ThemeProvider>
+        <div className="component-story-main">
+          <ThemeSwitcher />
+          <div
+            style={{
+              padding: '10px 0',
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center'
+            }}
+          >
+            <Story />
+          </div>
+        </div>
+      </ThemeProvider>
+    )
+  ]}
+/>
+
+# FileInput
+
+> Allows the user to drag and drop or browse one or several files.
+
+[![stability-unstable](https://img.shields.io/badge/stability-unstable-yellow.svg)](https://github.com/emersion/stability-badges#unstable)
+
+## Overview
+
+### Description
+
+`FileInput` allows the user to drag and drop or browse one or several files before upload these files for example.
+
+<Canvas>
+  <Story
+    name="Overview"
+    argTypes={{
+      description: {
+        control: false,
+        table: {
+          type: { summary: 'Node' }
+        }
+      },
+      acceptedFormats: {
+        control: false,
```

Comment:
Why disabling `acceptedFormats` control? 
Storybook can handle those [controls](https://storybook.js.org/docs/react/essentials/controls#configuration) by default, I'd like to let the user play with it :)